src/components/LocationDetails.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+
+import "../styles/LocationDetails.css";
+
+function LocationDetails({ errorMessage, city, country }) {
+  return errorMessage ? (
+    <section className="location-details">
+      <h1 className="location-details__heading">{errorMessage}</h1>
+    </section>
+  ) : (
+    <section className="location-details">
+      <h1 className="location-details__heading">{`${city}, ${country}`}</h1>
+    </section>
+  );
```

Comment:
Theres a bit of unneeded repetition in here. You're basically checking if the errorMessage is present and rending it if so, alternatively you're just adding the city and country to the same JSX structure. Why not something like this:
```
  return (
    <section className="location-details">
      <h1 className="location-details__heading">{errorMessage ? errorMessage : `${city}, ${country}`}</h1>
    </section>
  ) : (
    <section className="location-details">
      <h1 className="location-details__heading">{`${city}, ${country}`}</h1>
    </section>
  );
```

Reply:
Sorry I'm not sure if I follow here, isn't this exactly the same in terms of code repetition? I feel like in the case above the ternary's else clause should be `null` right? But in that case it makes me think there's a better way, using the `&&`?

Reply:
Actually, I just thought about it... how is this? 

```
  return (
    <section className="location-details">
      <h1 className="location-details__heading">
        {errorMessage || `${city}, ${country}`}
      </h1>
    </section>
  );
```